### DISCUSSION TITLE
Bloomz Model

### DISCUSSION THREAD

**Discussion by Alihkhawaher** [LostRuins/koboldcpp#249] [General]
Hello,

I am trying to run the following model

https://huggingface.co/rustformers/bloomz-ggml

But Koboldcpp freezes then crashed. 

![image](https://github.com/LostRuins/koboldcpp/assets/7887710/d299598b-c86f-4c7e-a99e-e7e7a92e9d67)

This happened with koboldcpp-1.30 and still with koboldcpp-1.31. I also tried with oobabooga. Anyone knows why I cannot load this model?

**Comment by LostRuins** [comment]
Bloomz is not supported at this time.
